### DISCUSSION TITLE
Does "Continuously trim servos on Fixed Wing" require a GPS?

### DISCUSSION THREAD

**Discussion by TTGG4400** [iNavFlight/inav#7168] [Q&A] [answered]
Does "Continuously trim servos on Fixed Wing" require a GPS?  I ask, because the INC help icon next to the selection says GPS required, but nowhere else is that indicated.

**Comment by JulioCesarMatias** [comment, marked as answer]
the GPS will be used to check if the fuselage is moving. The speed must be greater than or equal to 300 cm/s.

**Reply by avsaase** [reply]
This is to avoid adjusting the servo midpoints when the plane is not flying. Without this check, the servos would be trimmed when arming on the ground and rotating the model of moving the sticks. It is a bit of a crude check but we don't have better flying/not flying detection at this time.

**Reply by TTGG4400** [reply]
Thx - Think I'll add a GPS.

**Comment by JulioCesarMatias** [comment]
don't have it in other places because maybe they forgot to put it.
